lib/Command/Bot/Setup.php

```diff
@@ -81,6 +84,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			try {
 				$this->botConversationMapper->insert($bot);
 				$output->writeln('<info>Successfully set up for conversation ' . $token . '</info>');
+
+				$event = new BotEnabledEvent($room, $botServer);
```

Comment:
![grafik](https://github.com/nextcloud/spreed/assets/213943/8330dfc9-fc2d-4ffa-94af-7e28b7aeab53)


What we should do is `continue;` when `$returnCode = 2;`:
```diff
diff --git a/lib/Command/Bot/Setup.php b/lib/Command/Bot/Setup.php
index b0dcfd5f2..bb66955f5 100644
--- a/lib/Command/Bot/Setup.php
+++ b/lib/Command/Bot/Setup.php
@@ -70,10 +70,12 @@ class Setup extends Base {
                                if ($room->isFederatedConversation()) {
                                        $output->writeln('<error>Federated conversations can not have bots: ' . $token . '</error>');
                                        $returnCode = 2;
+                                       continue;
                                }
                        } catch (RoomNotFoundException) {
                                $output->writeln('<error>Conversation could not be found by token: ' . $token . '</error>');
                                $returnCode = 2;
+                               continue;
                        }
 
                        $bot = new BotConversation();
```